kubetest/gke.go

```diff
@@ -287,23 +290,28 @@ func (g *gkeDeployer) IsUp() error {
 //
 // TODO(zmerlynn): This whole path is really gross, but this seemed
 // the least gross hack to get this done.
+//
+// TODO(shyamjvs): Make this work with regional clusters where
```

Comment:
Is it only about regional clusters?
I think it's also about multizonal clusters (where you have single master, but nodes in different zones).

Reply:
Changed the comment. Though I'm not sure if we're running any gke tests that are multizonal except the regional one.
Federation has some tests on gce with clusters from different zones, but not on gke.